src/algoliasearch.js

```diff
@@ -120,6 +120,8 @@ var AlgoliaSearch = function(applicationID, apiKey, methodOrOptions, resolveDNS,
     }
 };
 
+AlgoliaSearch.JSONPCounter = 0;
```

Comment:
Could you add a comment to explain this? What about renaming it to JSONP_ID or something that has the semantic of an identifier?